src/compute-client/src/controller/instance.rs

```diff
@@ -411,9 +411,9 @@ impl<T: ComputeControllerTimestamp> Instance<T> {
 
     /// Enqueue the given response for delivery to the controller clients.
     fn deliver_response(&mut self, response: ComputeControllerResponse<T>) {
-        self.response_tx
-            .send(response)
-            .expect("global controller never drops");
+        if self.response_tx.send(response).is_err() {
```

Comment:
Why not just a `let _ = ...` for these, since we're not logging on error or anything. Feel free to ignore, because it's only cosmetic.

Reply:
No good reason I think. I just copied an existing pattern we already had before, but there we had the `is_err` because we wanted to emit a log. I'm deleting that log here, so the justification for `is_err` is gone now as well.

Reply:
Changed